sanitise-and-escape.php

```diff
@@ -0,0 +1,63 @@
+<?php 
+	
+namespace Sanitize_And_Escape_Me;
+
+class Results {
+	public function output_form(){
```

Comment:
You should also check to see if the form was submitted, and if so, call `output_results()`.